addons/account/models/account_account.py

```diff
@@ -34,7 +34,7 @@ def _check_account_type_unique_current_year_earning(self):
                 account_unaffected_earnings = self.browse(res['ids'])
                 raise ValidationError(_('You cannot have more than one account with "Current Year Earnings" as type. (accounts: %s)', [a.code for a in account_unaffected_earnings]))
 
-    name = fields.Char(string="Account Name", required=True, index='trigram', tracking=True)
+    name = fields.Char(string="Account Name", required=True, index='trigram', tracking=True, overrides={'l10n_multilang': fields.Char(translate=True)})
```

Comment:
It's cool, but what about custom modules that mark standard fields as translatable?
Do you have a solution in mind for the master branch?

Reply:
Do you mean modify code in module `l10n_multilang`?
I wish we could do that, but it is not possible because when module `account` is loaded for install/upgrade, the file for module `l10n_multilang` hasn't been loaded. In order to avoid column type change and mark the field `translate=True` we can only hack the code in the module `account`.

Reply:
I mean if a partner wants to mark another standard field as translatable how they could do that? Modify standard code?

Reply:
Ivan means situation, where some random module from apps store adds translation to other field.
Current implementation is crutch ("костыль" from Russian, a.k.a "workaround" in English) and does not fix the bug in general.

Reply:
We decide they cannot do that, this hack is only for the `l10n_multilang` and `account`.

The root problem is when install/upgrade a module A, module B which overrides module A is not loaded. As long as we don't change this, the problem cannot be perfectly solved.

Reply:
@em230418 better translation for Костыль is [Kludge](https://en.wikipedia.org/wiki/Kludge)
![image](https://user-images.githubusercontent.com/186131/213670321-c5460e52-34aa-4f16-9fe9-1233ba4f6b8a.png)

Reply:
Is there any chance to fix for the custom modules that mark standard fields as translatable? In some case, we want to add translation for standard fields for our requirements but losing translation and  need to reconfigure the translation every time we work on updating some module is really annoying.

Reply:
> Is there any chance to fix for the custom modules that mark standard fields as translatable? In some case, we want to add translation for standard fields for our requirements but losing translation and need to reconfigure the translation every time we work on updating some module is really annoying.

Sorry, I cannot promise it will be fixed. It is relevant to our upgrade flow, changing it may cause more unexpected problems. But we are working on it.
On the other hand, maybe you can discuss with our functional team. If a field has to be translated usually it should be translated in the standard code. It is a good timing for you and for us to think if a field really needs to be translated or not.